yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -381,16 +381,24 @@ func init() {
 		`character used as delimiter in rows of the table(s)(default is comma for CSV and tab for TEXT format)`)
 
 	importDataFileCmd.Flags().StringVar(&dataDir, "data-dir", "",
-		"path to the directory contains data files to import into table(s)")
+		"path to the directory contains data files to import into table(s)\n"+
```

Comment:
`path to the directory which contains the data files to be imported into the table(s)`